lib/portage/update.py

```diff
@@ -175,14 +175,11 @@ def grab_updates(updpath, prev_mtimes=None):
 		raise
 	if prev_mtimes is None:
 		prev_mtimes = {}
-	# validate the file name (filter out CVS directory, etc...)
-	mylist = [myfile for myfile in mylist if len(myfile) == 7 and myfile[1:3] == "Q-"]
+	# skip dotfiles
+	mylist = [myfile for myfile in mylist if not myfile.startswith(".")]
```

Comment:
Directories should also be skipped. The proposed spec says:
"All regular files in this directory will be processed, unless their name begins with a dot."